extensions/extensions-helm/README.md

```diff
@@ -27,52 +27,19 @@ Follow the instructions in the [_Kubernetes Artifacts Upgrade_](https://boozalle
-Helm](https://github.com/boozallen/aissemble/issues/488#issuecomment-2518466847), so we need to do some setup work to enable a plugin that provides this functionality.
-
-### Add the plugin to ArgoCD
-
-The repo server is responsible for running Helm commands to push changes into the cluster.  This is
-[documented](https://argo-cd.readthedocs.io/en/stable/user-guide/helm/#using-initcontainers) in ArgoCD, however these instructions didn't work, at least with our current chart
-version of 7.4.1. (Note there were some discussions on the `argocd-helm` GitHub about a specific update causing a breaking change in this functionality, and the latest docs were
-the "fix" for the breaking change. So could be that the old instructions would have worked fine.) To do this, we'll add an init container to the repo server that installs the
-plugin to a shared volume mount.
-
-```yaml
-aissemble-infrastructure-chart:
-  argo-cd:
-    repoServer:
-      env:
-        - name: HELM_CACHE_HOME
-          value: /helm-working-dir/.cache #Work around for install issue where plugins and cache locations being the same conflicts
-      initContainers:
-        - name: helm-plugin-install
-          image: alpine/helm
-          env:
-            - name: HELM_PLUGINS
-              value: /helm-working-dir/plugins #Configure Helm to write to the volume mount that the repo server uses
-          volumeMounts:
-            - mountPath: /helm-working-dir
-              name: helm-working-dir
-          command: [ "/bin/sh", "-c" ]
-          args: # install plugin
-            - apk --no-cache add curl;
-              helm plugin install https://github.com/aslafy-z/helm-git --version 1.3.0;
-              chmod -R 777 $HELM_PLUGINS;
-```
-
-### Updating the chart dependency
-
-To use your modified chart in the downstream project, the following changes should be made to the `Chart.yaml` file that pulls in the modified chart as a dependency:
-
- * Point `repository` to the modified chart on your branch in GitHub
-   * e.g.: `git+https://github.com/boozallen/aissemble/@extensions/extensions-helm/<modified-chart>?ref=<your-branch>`
-   * _**NB:** if the chart being tested is in a nested project under extensions-helm, update the repo path accordingly_
- * Set `version` to `1.0.0`
-
-### Potential pitfalls
-
- * There is an issue with committing Chart.lock files when using an explicit repository vs a repository alias, so Chart.lock files must not be committed.
+* When completing and locally testing a migration of a module to Extensions Helm, the above steps can be taken.
+  * As a precursor to the above steps, it would be helpful to create a simple aiSSEMBLE project to use as a test-bed.
+* It is important to note that because the module's helm charts will not be published to the Helm Repository remote 
+  until a build via GitHub Actions is completed, the `repository` field in the module's `Chart.yaml` file must be updated 
```

Comment:
Q: Is this a scenario where after the release and subsequent build in GHA then the workaround is no longer needed?  Just trying to understand the scenario.

Reply:
Yeah this is specifically to test chart changes pre-merge to dev. After that the changes would be available in ghcr.io as the lastest snapshot.

Because ArgoCD is strictly GitOps it required a special approach. Since we're not yet switching over to Argo for local, the old instructions apply.